2020-09-11 10:59:51.799 +02:00 [Error] LogItem { Timestamp: 09/11/2020 10:59:48, Message: "Invalid object name 'Article'.", NomApplication: "Arram", Layer: "Controleur Article", Location: null, MethodName: "Nom de la méthode : MoveNext Nom du fichier : C:\Projets\arram-back\arram-back\Arram.Core.API\Controllers\ArticleController.cs Numero de ligne : 68", Hostname: null, UserId: null, UserName: null, CustomerId: null, ClientId: 0, CustomerName: null, ElapsedMilliseconds: null, Exception: SqlException { Errors: [SqlError { Source: "Core Microsoft SqlClient Data Provider", Number: 208, State: 1, Class: 16, Server: "(localdb)\MSSQLLocalDB", Message: "Invalid object name 'Article'.", Procedure: "", LineNumber: 1 }], ClientConnectionId: e36b8040-5adf-49ad-8dc4-1b50cfca2d70, Class: 16, LineNumber: 1, Number: 208, Procedure: "", Server: "(localdb)\MSSQLLocalDB", State: 1, Source: "Core Microsoft SqlClient Data Provider", ErrorCode: -2146232060, TargetSite: System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader]), StackTrace: "   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.ExecuteImplementationAsync[TState,TResult](Func`4 operation, Func`4 verifySucceeded, TState state, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.ExecuteImplementationAsync[TState,TResult](Func`4 operation, Func`4 verifySucceeded, TState state, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Arram.Core.Repo.Repositories.ArticleRepository.GetAllActifAsync(CancellationToken ct) in C:\Projets\arram-back\arram-back\Arram.Core.Repo\Repositories\ArticleRepository.cs:line 50
   at Arram.Core.Business.ArticleLogic.ListeActifAsync() in C:\Projets\arram-back\arram-back\Arram.Core.Business\ArticleLogic.cs:line 52
   at Arram.Core.API.Controllers.ArticleController.Get(CancellationToken ct) in C:\Projets\arram-back\arram-back\Arram.Core.API\Controllers\ArticleController.cs:line 68", Message: "Invalid object name 'Article'.", Data: [DictionaryEntry { Key: "HelpLink.ProdName", Value: "Microsoft SQL Server" }, DictionaryEntry { Key: "HelpLink.ProdVer", Value: "13.00.4001" }, DictionaryEntry { Key: "HelpLink.EvtSrc", Value: "MSSQLServer" }, DictionaryEntry { Key: "HelpLink.EvtID", Value: "208" }, DictionaryEntry { Key: "HelpLink.BaseHelpUrl", Value: "http://go.microsoft.com/fwlink" }, DictionaryEntry { Key: "HelpLink.LinkId", Value: "20476" }], InnerException: null, HelpLink: null, HResult: -2146232060 }, CorrelationId: null, AdditionalInfo: null, AlerteEmail: False }
2020-09-11 11:16:54.799 +02:00 [Error] LogItem { Timestamp: 09/11/2020 11:16:47, Message: "Invalid object name 'Article'.", NomApplication: "Arram", Layer: "Controleur Article", Location: null, MethodName: "Nom de la méthode : MoveNext Nom du fichier : C:\Projets\arram-back\arram-back\Arram.Core.API\Controllers\ArticleController.cs Numero de ligne : 68", Hostname: null, UserId: null, UserName: null, CustomerId: null, ClientId: 0, CustomerName: null, ElapsedMilliseconds: null, Exception: SqlException { Errors: [SqlError { Source: "Core Microsoft SqlClient Data Provider", Number: 208, State: 1, Class: 16, Server: "(localdb)\MSSQLLocalDB", Message: "Invalid object name 'Article'.", Procedure: "", LineNumber: 1 }], ClientConnectionId: 7222c039-5264-41e8-b998-ea7bb228d2a8, Class: 16, LineNumber: 1, Number: 208, Procedure: "", Server: "(localdb)\MSSQLLocalDB", State: 1, Source: "Core Microsoft SqlClient Data Provider", ErrorCode: -2146232060, TargetSite: System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader]), StackTrace: "   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.ExecuteImplementationAsync[TState,TResult](Func`4 operation, Func`4 verifySucceeded, TState state, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.ExecuteImplementationAsync[TState,TResult](Func`4 operation, Func`4 verifySucceeded, TState state, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Arram.Core.Repo.Repositories.ArticleRepository.GetAllActifAsync(CancellationToken ct) in C:\Projets\arram-back\arram-back\Arram.Core.Repo\Repositories\ArticleRepository.cs:line 50
   at Arram.Core.Business.ArticleLogic.ListeActifAsync() in C:\Projets\arram-back\arram-back\Arram.Core.Business\ArticleLogic.cs:line 52
   at Arram.Core.API.Controllers.ArticleController.Get(CancellationToken ct) in C:\Projets\arram-back\arram-back\Arram.Core.API\Controllers\ArticleController.cs:line 68", Message: "Invalid object name 'Article'.", Data: [DictionaryEntry { Key: "HelpLink.ProdName", Value: "Microsoft SQL Server" }, DictionaryEntry { Key: "HelpLink.ProdVer", Value: "13.00.4001" }, DictionaryEntry { Key: "HelpLink.EvtSrc", Value: "MSSQLServer" }, DictionaryEntry { Key: "HelpLink.EvtID", Value: "208" }, DictionaryEntry { Key: "HelpLink.BaseHelpUrl", Value: "http://go.microsoft.com/fwlink" }, DictionaryEntry { Key: "HelpLink.LinkId", Value: "20476" }], InnerException: null, HelpLink: null, HResult: -2146232060 }, CorrelationId: null, AdditionalInfo: null, AlerteEmail: False }
2020-09-11 11:55:41.763 +02:00 [Error] LogItem { Timestamp: 09/11/2020 11:54:47, Message: "Invalid object name 'Article'.", NomApplication: "Arram", Layer: "Controleur Article", Location: null, MethodName: "Nom de la méthode : MoveNext Nom du fichier : C:\Projets\arram-back\arram-back\Arram.Core.API\Controllers\ArticleController.cs Numero de ligne : 68", Hostname: null, UserId: null, UserName: null, CustomerId: null, ClientId: 0, CustomerName: null, ElapsedMilliseconds: null, Exception: SqlException { Errors: [SqlError { Source: "Core Microsoft SqlClient Data Provider", Number: 208, State: 1, Class: 16, Server: "(localdb)\MSSQLLocalDB", Message: "Invalid object name 'Article'.", Procedure: "", LineNumber: 1 }], ClientConnectionId: bc269a01-cb86-4ca4-9140-7d94234f76d8, Class: 16, LineNumber: 1, Number: 208, Procedure: "", Server: "(localdb)\MSSQLLocalDB", State: 1, Source: "Core Microsoft SqlClient Data Provider", ErrorCode: -2146232060, TargetSite: System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader]), StackTrace: "   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.ExecuteImplementationAsync[TState,TResult](Func`4 operation, Func`4 verifySucceeded, TState state, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.ExecuteImplementationAsync[TState,TResult](Func`4 operation, Func`4 verifySucceeded, TState state, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Arram.Core.Repo.Repositories.ArticleRepository.GetAllActifAsync(CancellationToken ct) in C:\Projets\arram-back\arram-back\Arram.Core.Repo\Repositories\ArticleRepository.cs:line 50
   at Arram.Core.Business.ArticleLogic.ListeActifAsync() in C:\Projets\arram-back\arram-back\Arram.Core.Business\ArticleLogic.cs:line 52
   at Arram.Core.API.Controllers.ArticleController.Get(CancellationToken ct) in C:\Projets\arram-back\arram-back\Arram.Core.API\Controllers\ArticleController.cs:line 68", Message: "Invalid object name 'Article'.", Data: [DictionaryEntry { Key: "HelpLink.ProdName", Value: "Microsoft SQL Server" }, DictionaryEntry { Key: "HelpLink.ProdVer", Value: "13.00.4001" }, DictionaryEntry { Key: "HelpLink.EvtSrc", Value: "MSSQLServer" }, DictionaryEntry { Key: "HelpLink.EvtID", Value: "208" }, DictionaryEntry { Key: "HelpLink.BaseHelpUrl", Value: "http://go.microsoft.com/fwlink" }, DictionaryEntry { Key: "HelpLink.LinkId", Value: "20476" }], InnerException: null, HelpLink: null, HResult: -2146232060 }, CorrelationId: null, AdditionalInfo: null, AlerteEmail: False }
2020-09-11 12:07:53.805 +02:00 [Error] LogItem { Timestamp: 09/11/2020 12:07:52, Message: "Invalid column name 'AuteurId'.
Invalid column name 'TypeArticleId'.", NomApplication: "Arram", Layer: "Controleur Article", Location: null, MethodName: "Nom de la méthode : MoveNext Nom du fichier : C:\Projets\arram-back\arram-back\Arram.Core.API\Controllers\ArticleController.cs Numero de ligne : 68", Hostname: null, UserId: null, UserName: null, CustomerId: null, ClientId: 0, CustomerName: null, ElapsedMilliseconds: null, Exception: SqlException { Errors: [SqlError { Source: "Core Microsoft SqlClient Data Provider", Number: 207, State: 1, Class: 16, Server: "(localdb)\MSSQLLocalDB", Message: "Invalid column name 'AuteurId'.", Procedure: "", LineNumber: 1 }, SqlError { Source: "Core Microsoft SqlClient Data Provider", Number: 207, State: 1, Class: 16, Server: "(localdb)\MSSQLLocalDB", Message: "Invalid column name 'TypeArticleId'.", Procedure: "", LineNumber: 1 }], ClientConnectionId: cef3911b-c32b-4ce1-87f6-05257f561326, Class: 16, LineNumber: 1, Number: 207, Procedure: "", Server: "(localdb)\MSSQLLocalDB", State: 1, Source: "Core Microsoft SqlClient Data Provider", ErrorCode: -2146232060, TargetSite: System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader]), StackTrace: "   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.ExecuteImplementationAsync[TState,TResult](Func`4 operation, Func`4 verifySucceeded, TState state, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.ExecuteImplementationAsync[TState,TResult](Func`4 operation, Func`4 verifySucceeded, TState state, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Arram.Core.Repo.Repositories.ArticleRepository.GetAllActifAsync(CancellationToken ct) in C:\Projets\arram-back\arram-back\Arram.Core.Repo\Repositories\ArticleRepository.cs:line 50
   at Arram.Core.Business.ArticleLogic.ListeActifAsync() in C:\Projets\arram-back\arram-back\Arram.Core.Business\ArticleLogic.cs:line 52
   at Arram.Core.API.Controllers.ArticleController.Get(CancellationToken ct) in C:\Projets\arram-back\arram-back\Arram.Core.API\Controllers\ArticleController.cs:line 68", Message: "Invalid column name 'AuteurId'.
Invalid column name 'TypeArticleId'.", Data: [DictionaryEntry { Key: "HelpLink.ProdName", Value: "Microsoft SQL Server" }, DictionaryEntry { Key: "HelpLink.ProdVer", Value: "13.00.4001" }, DictionaryEntry { Key: "HelpLink.EvtSrc", Value: "MSSQLServer" }, DictionaryEntry { Key: "HelpLink.EvtID", Value: "207" }, DictionaryEntry { Key: "HelpLink.BaseHelpUrl", Value: "http://go.microsoft.com/fwlink" }, DictionaryEntry { Key: "HelpLink.LinkId", Value: "20476" }], InnerException: null, HelpLink: null, HResult: -2146232060 }, CorrelationId: null, AdditionalInfo: null, AlerteEmail: False }
2020-09-11 12:11:13.570 +02:00 [Error] LogItem { Timestamp: 09/11/2020 12:11:12, Message: "Invalid column name 'TypeArticleId'.", NomApplication: "Arram", Layer: "Controleur Article", Location: null, MethodName: "Nom de la méthode : MoveNext Nom du fichier : C:\Projets\arram-back\arram-back\Arram.Core.API\Controllers\ArticleController.cs Numero de ligne : 68", Hostname: null, UserId: null, UserName: null, CustomerId: null, ClientId: 0, CustomerName: null, ElapsedMilliseconds: null, Exception: SqlException { Errors: [SqlError { Source: "Core Microsoft SqlClient Data Provider", Number: 207, State: 1, Class: 16, Server: "(localdb)\MSSQLLocalDB", Message: "Invalid column name 'TypeArticleId'.", Procedure: "", LineNumber: 1 }], ClientConnectionId: 914ec900-4742-4562-97aa-8e00ac36d386, Class: 16, LineNumber: 1, Number: 207, Procedure: "", Server: "(localdb)\MSSQLLocalDB", State: 1, Source: "Core Microsoft SqlClient Data Provider", ErrorCode: -2146232060, TargetSite: System.Data.Common.DbDataReader <ExecuteDbDataReaderAsync>b__164_0(System.Threading.Tasks.Task`1[Microsoft.Data.SqlClient.SqlDataReader]), StackTrace: "   at Microsoft.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__164_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReaderAsync(RelationalCommandParameterObject parameterObject, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.InitializeReaderAsync(DbContext _, Boolean result, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.ExecuteImplementationAsync[TState,TResult](Func`4 operation, Func`4 verifySucceeded, TState state, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.ExecutionStrategy.ExecuteImplementationAsync[TState,TResult](Func`4 operation, Func`4 verifySucceeded, TState state, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Query.Internal.QueryingEnumerable`1.AsyncEnumerator.MoveNextAsync()
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.EntityFrameworkQueryableExtensions.ToListAsync[TSource](IQueryable`1 source, CancellationToken cancellationToken)
   at Arram.Core.Repo.Repositories.ArticleRepository.GetAllActifAsync(CancellationToken ct) in C:\Projets\arram-back\arram-back\Arram.Core.Repo\Repositories\ArticleRepository.cs:line 50
   at Arram.Core.Business.ArticleLogic.ListeActifAsync() in C:\Projets\arram-back\arram-back\Arram.Core.Business\ArticleLogic.cs:line 52
   at Arram.Core.API.Controllers.ArticleController.Get(CancellationToken ct) in C:\Projets\arram-back\arram-back\Arram.Core.API\Controllers\ArticleController.cs:line 68", Message: "Invalid column name 'TypeArticleId'.", Data: [DictionaryEntry { Key: "HelpLink.ProdName", Value: "Microsoft SQL Server" }, DictionaryEntry { Key: "HelpLink.ProdVer", Value: "13.00.4001" }, DictionaryEntry { Key: "HelpLink.EvtSrc", Value: "MSSQLServer" }, DictionaryEntry { Key: "HelpLink.EvtID", Value: "207" }, DictionaryEntry { Key: "HelpLink.BaseHelpUrl", Value: "http://go.microsoft.com/fwlink" }, DictionaryEntry { Key: "HelpLink.LinkId", Value: "20476" }], InnerException: null, HelpLink: null, HResult: -2146232060 }, CorrelationId: null, AdditionalInfo: null, AlerteEmail: False }
2020-09-11 12:12:30.370 +02:00 [Error] LogItem { Timestamp: 09/11/2020 12:12:29, Message: "Pas trouvé 99999", NomApplication: "Arram", Layer: "Controleur Licence", Location: null, MethodName: "Nom de la méthode : MoveNext Nom du fichier : C:\Projets\arram-back\arram-back\Arram.Core.API\Controllers\LicenceController.cs Numero de ligne : 50", Hostname: null, UserId: null, UserName: null, CustomerId: null, ClientId: 0, CustomerName: null, ElapsedMilliseconds: null, Exception: null, CorrelationId: null, AdditionalInfo: null, AlerteEmail: False }
